src/components/Person/Person.jsx

```diff
@@ -1 +1,21 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">{`My name is ${person.name}`}</h2>
+
+    {
+      person.age && (
+        <p className="Person__age">
+          { `I am ${person.age}` }
+        </p>
+      )
+    }
+
+    <p className="Person__partner">
+      {
+      person.isMarried
+        ? `${person.partnerName} is my ${person.sex === 'm' ? 'wife' : 'husband'}`
+        : 'I am not married'
```

Comment:
Create a variable for husband/wife to make your code a bit cleaner
![image](https://github.com/mate-academy/react_person/assets/96409563/21dd777a-6206-4e79-a6bb-00aabc15f1cb)